rust/datafusion/src/execution/aggregate.rs

```diff
@@ -1465,7 +1465,8 @@ mod tests {
     }
 
     fn load_csv(filename: &str, schema: &Arc<Schema>) -> Rc<RefCell<dyn Relation>> {
-        let ds = CsvBatchIterator::new(filename, schema.clone(), true, &None, 10);
+        let ds =
+            CsvBatchIterator::try_new(filename, schema.clone(), true, &None, 10).unwrap();
```

Comment:
Shouldn't you propagate this error instead of using `unwrap`, this is the error from `File::open` so it's reasonable that load function would have to deal with this, including `load_csv`.

If there is a strong reason why you don't want to maybe just switch to `expect`.

Also, what are you not exposing `has_header`, etc.?  In particular, why is `10` used as the batch size?

Reply:
This is test code to support the unit tests and not part of the actual product ... but you are right, it would be better to have this method return a Result. I will fix that tonight.

Reply:
> This is test code to support the unit tests and not part of the actual product

Ahh, ok sorry.  It's hard to see the context while reviewing on github.  No need to change if it's test code.